kolibri/plugins/epub_viewer/assets/src/views/EpubRendererIndex.vue

```diff
@@ -333,6 +337,14 @@
         const seconds = (numberOfWords * 60) / WORDS_PER_MINUTE;
         return seconds;
       },
+      /* eslint-disable kolibri/vue-no-unused-properties */
+      /**
+       * @public
+       */
+      defaultDuration() {
+        return this.expectedTimeToRead;
```

Comment:
Is `expectedTimeToRead` being used anywhere else now? Could just move its implementation into here if not.